sdk/identity/identity/TROUBLESHOOTING.md

```diff
@@ -470,21 +470,14 @@ If the preceding command isn't working properly, follow the instructions to reso
+| AADSTS900023: Specified tenant identifier '975f013f-7f24-47e8-a7d3-abc4752bf34' is neither a valid DNS name, nor a valid external domain.                                                                                                                                                                                                                                | The tenant id passed to the credential is invalid                                                                                                                                 | Verify the tenant id is valid. If the service connection was configured via a user-assigned managed identity, the tenant will be the one in which managed identity was registered. If the service connection is configured via a Service Principal, the tenant should be the one in which the Service Principal is registered.                                                                             |
+| No service connection found with identifier <GUID>                                                                                                                                                                                                                                                                                                                       | The service connection id provided is incorrect.                                                                                                                                  | Verify the serviceConnectionId provided. This parameter refers to the `resourceId` of the Azure Service Connection. It can also be found in the queryString of the respective Service Connection's configuration page in Azure Devops.                                                                                                                                                                     |
+| something                                                                                                                                                                                                                                                                                                                                                                | somehting                                                                                                                                                                         | System Access Token is a required System variable in the Azure Pipeline task and [should be provided in the pipeline task as mentioned in the docs](https://learn.microsoft.com/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#systemaccesstoken).                                                                                                                                     |
```

Comment:
There's an extra row here, probably a placeholder that needs to be removed - specifically, the row that just says "something"